pimd/pim_nht.c

```diff
@@ -728,8 +728,8 @@ int pim_parse_nexthop_update(ZAPI_CALLBACK_ARGS)
 	pim = vrf->info;
 
 	if (!zapi_nexthop_update_decode(zclient->ibuf, &nhr)) {
-		zlog_err("%s: Decode of nexthop update from zebra failed",
-			 __func__);
+		zlog_warn("%s: Decode of nexthop update from zebra failed",
```

Comment:
Why is this a warn -vs- error?  This is an error that has happened that means there is a bug in the code that needs to be handled